### DISCUSSION TITLE
What package to use for CosmosDb

### DISCUSSION THREAD

**Discussion by vandenbergjp** [Azure/azure-cosmos-dotnet-v3#2617] [Q&A] [answered]
Looking at this page:
https://azure.github.io/azure-sdk/releases/latest/all/dotnet.html

I see that there is a `Microsoft.Azure.Cosmos `package which is updated relatively recent. However is says that it is replaced by Azure.Cosmos which is in preview and not updated for a long time.

Now I'm a bit confused. What package am I supposed to use?

**Comment by justinbhopper** [comment, marked as answer]
Azure.Cosmos appears to be the new package for v4, which is still in preview.  Use Microsoft.Azure.Cosmos for now

**Reply by j82w** [reply]
The v3 SDK which is [Microsoft.Azure.Cosmos](https://www.nuget.org/packages/Microsoft.Azure.Cosmos/3.20.1 ) is the current SDK and the one recommended for all production scenarios. A prototype of the v4 SDK(Azure.Cosmos) was done as a proof of concept. Currently there is no time frame on when the v4 SDK will be released. The amount of work required is not trivial.
